ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -614,6 +619,7 @@ private final class PostgresQueries(tablePrefix: String) extends Queries(tablePr
   protected[this] override def agreementTextType = sql"TEXT NOT NULL"
 
   protected[this] override def jsonColumn(name: Fragment) = name ++ sql" JSONB NOT NULL"
+  protected[this] override def keyHashColumn = textType
```

Comment:
Nitpick: a `val` should do here, right?

Reply:
I have to oppose this nitpick; you aren't saving any real work, and are signing up for potential initialization order problems. Better to keep it a `def`.